dlt/common/normalizers/json/relational.py

```diff
@@ -261,6 +258,17 @@ def extend_schema(self) -> None:
             }
         )
 
+    def extend_table(self, table_name: str) -> None:
```

Comment:
should we call it in "extend_schema"? ie. someone is modifying files directly and adding tables